test/unit/period/generators/bi-monthly_spec.js

```diff
@@ -35,27 +35,27 @@ describe('Bi-monthly period', () => {
                 startDate: '2017-03-01',
                 endDate: '2017-04-30',
                 name: `March - April 2017`,
-                id: '201703B',
+                id: '201702B',
```

Comment:
It doesn't help that the documentation doesn't actually explain how BiMonthly period types work. I had to reverse engineer them by looking at the analytics apps.. 🙄